internal/services/hybridkubernetes/registration.go

```diff
@@ -0,0 +1,29 @@
+package hybridkubernetes
+
+import "github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+
+type Registration struct{}
+
+// Name is the name of this Service
+func (r Registration) Name() string {
+	return "HybridKubernetes"
+}
+
+// WebsiteCategories returns a list of categories which can be used for the sidebar
+func (r Registration) WebsiteCategories() []string {
+	return []string{
+		"HybridKubernetes",
+	}
+}
+
+// SupportedDataSources returns the supported Data Sources supported by this Service
+func (r Registration) SupportedDataSources() map[string]*pluginsdk.Resource {
+	return map[string]*pluginsdk.Resource{}
+}
+
+// SupportedResources returns the supported Resources supported by this Service
+func (r Registration) SupportedResources() map[string]*pluginsdk.Resource {
+	return map[string]*pluginsdk.Resource{
+		"azurerm_hybrid_kubernetes_connected_cluster": resourceHybridKubernetesConnectedCluster(),
```

Comment:
per the product name, should this be:

```suggestion
		"azurerm_arc_kubernetes_cluster": resourceArcKubernetesCluster(),
```